Tools/scripts/summarize_stats.py

```diff
@@ -599,7 +599,8 @@ def output_stats(inputs, json_output=None):
     if len(inputs) == 1:
         stats = gather_stats(inputs[0])
         if json_output is not None:
-            json.dump(stats, json_output)
+            with json_output as json_output:
```

Comment:
I don't quite like this variable name shadowing. Maybe `as output_file`?

Reply:
Thanks :)